frontend/src/containers/AdmissionsContainer/Columns.jsx

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+import Icon from "src/components/Icon";
+import FormatTime from "src/components/Time/FormatTime";
+
+export const cols = [
+  {
+    id: "expander",
+    Header: ({ getToggleAllRowsExpandedProps, isAllRowsExpanded }) => (
+      <span {...getToggleAllRowsExpandedProps()}>
+        {isAllRowsExpanded ? "▲" : "▼"}
+      </span>
+    ),
+    Cell: ({ row }) => (
+      <span {...row.getToggleRowExpandedProps()}>
+        {row.isExpanded ? "▲" : "▼"}
+      </span>
+    ),
```

Comment:
Personally I'd prefer the not-expanded icon to be an arrow to the right instead of upwards